<!-- %Tag(FULL)%-->
<launch>


 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="xbox_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.35" />
    <param name="autorepeat_rate" value="20" />
    <remap from="/joy" to="/vehicle_joy" />
  </node>

 <!-- Axes -->
  <param name="X" value="0" type="int"/>
  <param name="XDir" value="1" type="int"/>
  <param name="Y" value="1" type="int"/>
  <param name="YDir" value="1" type="int"/>
  <param name="Z" value="4" type="int"/>
  <param name="ZDir" value="-1" type="int"/>
  <param name="Yaw" value="3" type="int"/>
  <param name="YawDir" value="1" type="int"/>
  <param name="RollDir" value="1" type="int"/>
  <param name="PitchDir" value="1" type="int"/>
  <param name="linear_scale" value="10" type="double"/>
  <param name="angular_scale" value="10" type="double"/>
  
  <param name="topic" value="/commands" type="string" />

  <node pkg="arm5_joy_control" type="vehicle_cartesian_control" name="vehicle_cartesian_control" respawn="false" output="screen"/>

</launch>
<!-- %EndTag(FULL)%-->
